lib/twitter-ads/campaign.rb

```diff
@@ -9,7 +9,7 @@ class Campaign
 
     attr_reader :account
 
-    property :id, read_only: true
+    property :id
```

Comment:
Allowing `:id` to be writeable will interfere with `save()` logic. You probably don't want to do this even in your own custom version. Rather than manually setting the `:id`, you should just load the object by `:id` from the API.
```ruby
# how to load a specific campaign
campaign_id = '3to3us'
campaign = account.campaigns(campaign_id)

# alternative and equivalent syntax
campaign_id = '3to3us'
campaign = TwitterAds::Campaign.load(account, campaign_id)
```